### DISCUSSION TITLE
Native WooCommerce Integration for magic links

### DISCUSSION THREAD

**Discussion by alpha-pix** [snicco/fortress#7] [Feature Requests]
Natively offer magic links in WooCommerce pages, and redirect the user to the same spot (eg. checkout) on login.
![59ae2438e2f7688353bf0ee802bcaa7d77f67470](https://github.com/snicco/fortress/assets/54539546/963c9847-44ad-461c-8e9f-d6854c7bd06a)

**Comment by calvinalkan** [comment]
Right now, you could do:

```php
add_filter('woocommerce_checkout_login_message', function (string $message) {
return "$message" . do_shortcode('[snicco-fortress:request_magic_link]Request a Login Link[/snicco-fortress:request_magic_link]');
});
```

Which would show as:
![screenshot-wp test-2023 08 09-20_28_02](https://github.com/snicco/fortress/assets/76868662/93d7d8ce-79b9-4ef3-8098-998021492163)

But the above does not work over AJAX and makes the user leave the checkout.

This has to wait until we have an AJAX based backend for logins.
